changelogs/fragments/4662-yarn-emoji.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

So either change this:
```suggestion
breaking_changes:
```
or add a version detection to the code. (From what you wrote, I don't think it's worth to invest time implementing a version detection...)

---

Time is a bit pressuring since the 5.0.0 release is coming up on Tuesday. If nobody complains I'll commit this suggestion and then merge this PR on Tuesday morning.